doc/releases/changelog-dev.md

```diff
@@ -38,6 +38,14 @@
 * Observable validation for `default.qubit` is now based on execution mode (analytic vs. finite shots) and measurement type (sample measurement vs. state measurement).
   [(#5890)](https://github.com/PennyLaneAI/pennylane/pull/5890)
 
+<h4>Community contributions 🥳</h4>
+
+* `~.DefaultQutritMixed` readout error has been added using parameters `readout_relaxation_probs` and 
```

Comment:
```suggestion
* `DefaultQutritMixed` readout error has been added using parameters `readout_relaxation_probs` and 
```
Note that we skip these relative links because it [doesn't work in the docs](https://xanaduai-pennylane--5842.com.readthedocs.build/en/5842/development/release_notes.html), or the plain markdown version.